__init__.py

```diff
@@ -1914,6 +1914,12 @@ class BlenderKitAddonPreferences(AddonPreferences):
         update=utils.save_prefs,
     )
 
+    show_VIEW3D_MT_blenderkit_model_properties: bpy.props.BoolProperty(
```

Comment:
Ugly but I think it makes sense. This way it is easier to search what this property affects. Would like to use propose this format for other properties also (those which are tied with mostly one object). WDYT @vilemduha?